File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int X=0;
        for(String i: operations){
            if(i.equals("X++") || i.equals("++X")){
                X++;
            }
            else if(i.equals("X--") || i.equals("--X")){
                X--;
            }
        }
        
        return X;

    }
}

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.py
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        X=0
        for i in operations:
            if "++" in i:
                X+=1
            elif "--" in i:
                X-=1
        return X
            
        #  if(i in ("X++","++X")):
        #         X+=1
        #     elif(i in ("X--","--X")):
        #         X-=1
        # return X
        
        